scripts/lib/install

```diff
@@ -14,16 +14,17 @@ DEPLOYMENT_TYPE="${DEPLOYMENT_TYPE:-voyager}"
 # Docker.  Use e.g. zulip::app_frontend for a Zulip frontend server.
 PUPPET_CLASSES="${PUPPET_CLASSES:-zulip::voyager}"
 VIRTUALENV_NEEDED="${VIRTUALENV_NEEDED:-yes}"
+ZULIP_ROOT="$(realpath $(dirname $0)/../../)"
```

Comment:
Pushed a version fixing that.  It also looks like we need to install `realpath`, since it isn't available in `coreutils` in Ubuntu Trusty.